packages/api/portfolios/environments/patchEnvironment.ts

```diff
@@ -0,0 +1,67 @@
+import "reflect-metadata";
+import { createConnection } from "../../utils/database";
+import { Portfolio } from "../../../orm/entity/Portfolio";
+import { Application } from "../../../orm/entity/Application";
+import { Environment, IEnvironmentOperators } from "../../../orm/entity/Environment";
+import { EnvironmentRepository } from "../../repository/EnvironmentRepository";
+import { APIGatewayProxyResult, Context } from "aws-lambda";
+import { ApiSuccessResponse, SuccessStatusCode } from "../../utils/response";
+import internalSchema = require("../../models/internalSchema.json");
+import middy from "@middy/core";
+import xssSanitizer from "../../portfolioDrafts/xssSanitizer";
+import jsonBodyParser from "@middy/http-json-body-parser";
+import validator from "@middy/validator";
+import JSONErrorHandlerMiddleware from "middy-middleware-json-error-handler";
+import cors from "@middy/http-cors";
+import { ApiGatewayEventParsed } from "../../utils/eventHandlingTool";
+import { CORS_CONFIGURATION } from "../../utils/corsConfig";
+import { wrapSchema } from "../../utils/schemaWrapper";
+import { errorHandlingMiddleware } from "../../utils/errorHandlingMiddleware";
+import { IpCheckerMiddleware } from "../../utils/ipLogging";
+import { validateRequestShape } from "../../utils/shapeValidator";
+
+/**
+ * Submits an update to an environment of an application
+ *
+ * @param event - The PUT request from API Gateway
```

Comment:
The PATCH operation is used to only update the operators, not the entire environment.

Reply:
Corrected, I forgot to update the JS doc. Thanks!